src/renderer/kube-object.store.ts

```diff
@@ -306,51 +306,31 @@ export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemSt
     });
   }
 
-  getSubscribeApis(): KubeApi[] {
-    return [this.api];
-  }
-
-  subscribe(apis = this.getSubscribeApis()) {
```

Comment:
So we decided to change KubeObjectStore subscibe api in this PR? Why?

Reply:
Because all stores now have only one API that they know about/need to know about. Since there are now separate stores for the `ClusterRoles` and `ClusterRoleBindings`.

If you really want I can move that to a separate PR, but I felt that it made sense to do it in the same PR that allowed for it to happen.

Reply:
should we also modify signature of `ItemStore.subscribe()` as well? (separate PR is fine if needed)